coderd/database/dbgen/dbgen.go

```diff
@@ -1195,17 +1195,6 @@ func Preset(t testing.TB, db database.Store, seed database.InsertPresetParams) d
 	return preset
 }
 
-func PresetParameter(t testing.TB, db database.Store, seed database.InsertPresetParametersParams) []database.TemplateVersionPresetParameter {
```

Comment:
not needed anymore?

Reply:
there were a few potential uses for it, where we were just using the db directly. added back.